templates/Dockerfile.j2

```diff
@@ -19,13 +19,8 @@ ENV PATH={{ beat_home }}:$PATH
 COPY config/{{ beat }}.yml {{ config_file }}
 
 # Provide a non-root user.
-RUN addgroup --gid 1000 {{ beat }} && \
-    adduser --disabled-password \
-            --gecos={{ beat }} \
-            --uid=1000 \
-            --gid=1000 \
-            --home {{ beat_home }} \
-            {{ beat }}
+RUN groupadd --gid 1000 {{ beat }} && \
+    adduser -M --uid 1000 --gid 1000 --home {{ beat_home }} {{ beat }}
```

Comment:
Being totally pedantic here that as I see you've switched from `addgroup` to `groupadd` similarly you could use the `useradd` for consistency. At any rate on RHEL platforms adduser is a symlink to that:

```
# ls -ld /usr/sbin/adduser 
lrwxrwxrwx 1 root root 7 Apr  6 14:52 /usr/sbin/adduser -> useradd
```